draft-fossati-tls-attestation.md

```diff
@@ -154,74 +154,108 @@ Several formats for encoding evidence are available, such as:
 - the Android Key Attestation, and
 - Apple Key Attestation. 
 
-Likewise, there are different encodings available for attestation results.
-One such encoding, AR4SI {{?I-D.ietf-rats-ar4si}} is being standardized by the RATS 
+Likewise, there are different encodings available for attestation results. One
+such encoding, AR4SI {{?I-D.ietf-rats-ar4si}} is being standardized by the RATS
 working group.
 
-This version of the specification defines how to support the background check model
-in  the TLS handshake, such that the details about the attestation technology are
-agnostic to the TLS handshake itself. Later versions of the specification will 
-support the passport model as well. 
+This specification defines how to support the background check model in the TLS
+handshake, such that the details about the attestation technology are agnostic
+to the TLS handshake itself. Similarly, support for the passport model is
```

Comment:
I would rephrase it as follows.

This specification supports both the background check and passport model, during TLS Handshake.

In background check model the details about the attestation technology are agnostice to TLS handshake itself.
Similarly, in the passport model, the details about the attestation results encoding and trust relationships are agnostic to  the TLS handshake.